pandas/core/indexes/base.py

```diff
@@ -324,6 +330,9 @@ class Index(IndexOpsMixin, PandasObject):
 
     >>> pd.Index(list('abc'))
     Index(['a', 'b', 'c'], dtype='object')
+
+    >>> pd.Index([1, 2, 3], dtype="uint8")
+    NumericIndex([1, 2, 3], dtype='uint8')
```

Comment:
This has to be updated later? (when NumericIndex is gone?)

Reply:
Yes, I`ll do a ctrl-f to clean up any cruft left at the end.